### ISSUE TITLE
[react-render-tracker] Error: Could not find ID

### ISSUE THREAD

**Issue by nickshevr** [lahmatiy/react-render-tracker#12]
**React v16.8.4**

Minimal trace:

```
react_devtools_backend.js:4049 [react-render-tracker] Error: Could not find ID for Fiber "Provider"
    at $ (browser.rtt.js:44)
    at fe (browser.rtt.js:44)
    at fe (browser.rtt.js:44)
    at fe (browser.rtt.js:44)
    at Object.tn [as handleCommitFiberRoot] (browser.rtt.js:44)
    at Object.onCommitFiberRoot (browser.rtt.js:3)
    at react-with-dom.js:13097
    at react-with-dom.js:13062
    at onCommitRoot (react-with-dom.js:13114)
    at commitRoot (react-with-dom.js:22229)
```

`browser.rtt.js` - js from dist as is.
I have multiple react root on my page.

**Comment by nickshevr** [comment]
Also I have **RC** with code like these:
```
 const WrappedProvider = (props: Props) => <Provider {...props} />;
WrappedProvider.displayName = `Provider(${name})`;
```

**Comment by nickshevr** [comment]
Think I have some cycle Oo
The function fails on this arg.
![image](https://user-images.githubusercontent.com/16756393/132924550-811fb166-d5d2-4c84-a66c-fd3e9aa0b58c.png)

**Comment by lahmatiy** [comment]
Thank you for reporting the issue!
It's hard to say what wrong in your case. It would be useful if you could provide a code for reproducing case. The source fragment with `WrappedProvider` you're provided looks like works well (here is [my try to reproduce the issue](https://codesandbox.io/s/zealous-firefly-ksr5r?file=/src/index.js)).
`FiberNode.alternate` is a circular linked list, that's not a problem because never iterated by the tool.

**Comment by lahmatiy** [comment]
@nickshevr Is <Suspence> used in your application? If so, probably I found a problem place. I can't reproduce it all the time, but it's suspence related for sure.

**Comment by Mavrin** [comment]
https://9lx9r.csb.app/ also should be enabled `react dev tools`.

**Comment by lahmatiy** [comment]
I fixed several bugs related to the issue. I've just published a new [version 0.4](https://github.com/lahmatiy/react-render-tracker/releases/tag/v0.4.0) with those fixes (and new cool features). I believe it should help.
Closing the issue for now. Feel free to reopen it if problem is still exist.
